mkosi/README.md

```diff
@@ -26,6 +26,9 @@ which requires VDI images.  You can generate it with this command:
 Please follow all the steps described in the LilyPond Contributor guide to
 [install and configure the VirtualBox machine](http://lilypond.org/doc/v2.19/Documentation/contributor/lilydev#installing-lilydev-in-virtualbox).
 In particular, remember that you need to enable EFI or the image won't boot.
+With the current legacy version of VirtualBox guest additions from
```

Comment:
BTW, do you think that upgrading debian images from stretch to buster could help with this?
See [virtualbox-guest* packages in Debian](https://packages.debian.org/search?keywords=virtualbox-guest).